crates/driver/src/domain/eth/mod.rs

```diff
@@ -261,6 +261,13 @@ impl From<H160> for WethAddress {
     }
 }
 
+#[allow(clippy::from_over_into)]
```

Comment:
Added this because doing `Impl X for H160` being `H160` an external type is not a good idea. And we only need the `Into` anyway.